app/web/markdown/blog/2025/04/20/couchers-inc-financials-2024.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Couchers, Inc. financials for 2024
+slug: couchers-inc-financials-2024
+description: Summary of the financials of Couchers, Inc. for 2024
+date: 2025/04/20
+author: Aapeli
+---
+
+This is a very short summary of the 2024 financials of Couchers, Inc.: the 501(c)3 non-profit that supports Couchers.org.
+
+**Couchers is aiming to raise $5000 in 2025** to pay for hosting the platform, and supporting the development and growth of the community. This is the **less than 10 cents per user**. We'll reach this goal if just **one in a thousand users chip in with $10!**
+
+## What we spent money on
+
+Couchers spent **$1948.26** in 2024, equivalent to **4.1 cents per user**, the breakdown of costs is as follows:
+
+* Amazon Web Services (AWS) &mdash; **$1524.64** (78.3%): hosting of the Couchers.org platform and development/support tools
+* Calendly &mdash; **$162.00** (8.3%): meeting scheduling platform we use to organize volunteer onboarding calls
+* Zoom &mdash; **$159.89** (8.2%): videoconferencing platform for volunteer meetings
+* Expo.dev &mdash; **$39.00** (2.0%): a development tool to automatically build and deploy preview versions of the upcoming native app
+* Porkbun &mdash; **$37.73** (1.9%): domain registrar
+* Google Play &mdash; **$25.00** (1.3%): signing up for a Google Play account to distribute the upcoming Android app
+
+![Pie chart of our 2024 expenses](/img/blog/20250420_expenses_pie_chart.png)
+
+## Where we received money from
+
+Couchers received **$1257.13** via donations from the community after Stripe fees (our payment processor). We processed **$1345.01** of donations and paid fees of **$58.72** in 2024 (note that there is a slight discrepancy to what showed up in our bank account in 2024 due to a few day delay in Stripe paying out donations). That is, we paid 4.4% in payment processing fees. Larger donations have a lower per-transaction fee (if you would like to make a large donation, we can issue a one-off invoice for a bank transfer that has practically no fee).
```

Comment:
Would it make sense to put the payment processing fees in the "What we spent money on" and the pie chart?

Reply:
I agree with Nicole here since those fees are part of our OpEx

Reply:
Resolving as discussed